docs/supported_instruments.rst

```diff
@@ -0,0 +1,105 @@
+Supported Instruments
+=====================
+
+C/NOFS IVM
+----------
+
+.. automodule:: pysatNASA.instruments.cnofs_ivm
+   :members:
+
+C/NOFS PLP
+----------
+
+.. automodule:: pysatNASA.instruments.cnofs_plp
+   :members:
+
+C/NOFS VEFI
+-----------
+
+.. automodule:: pysatNASA.instruments.cnofs_vefi
+   :members:
+
+DE2 LANG
+-----------
+
+.. automodule:: pysatNASA.instruments.de2_lang
+   :members:
+
+DE2 NACS
```

Comment:
Recommend adding labels so that Constellations can reference the included instruments.